source/gems/dry-validation/nested-data.html.md

```diff
@@ -89,3 +89,30 @@ puts errors.inspect
 #   }
 # }
 ```
+
+Similarly, you use `each` and `schema` to validate an array of hashes:
+
+``` ruby
+schema = Dry::Validation.Schema do
+  required(:people).each do
+    schema do
+      required(:name).filled(:str?)
+      required(:age).filled(:int?, gteq?: 18)
+    end
+  end
+end
+
+errors = schema.call(
+  people: [ { name: 'Alice', age: 19 }, { name: 'Bob', age: 17 } ],
+).messages
+
+errors = schema.call(phone_numbers: ['123456789', 123456789]).messages
```

Comment:
I can do, but on closer inspection I've realised: is there any functional difference between these two schemas?

```
Dry::Validation.Schema do
  required(:people).each do
    required(:name).filled(:str?)
    required(:age).filled(:int?, gteq?: 18)
  end
end

Dry::Validation.Schema do
  required(:people).each do
    schema do
      required(:name).filled(:str?)
      required(:age).filled(:int?, gteq?: 18)
    end
  end
end
```

Are these both approaches that should be documented?